src/param.rs

```diff
@@ -316,19 +320,20 @@ impl AudioNode for AudioParam {
 impl AudioParam {
     /// Current value of the automation rate of the AudioParam
     pub fn automation_rate(&self) -> AutomationRate {
-        self.raw_parts.shared_parts.load_automation_rate()
+        self.raw_parts.automation_rate
     }
 
     /// Update the current value of the automation rate of the AudioParam
     ///
     /// # Panics
     ///
     /// Some nodes have automation rate constraints and may panic when updating the value.
-    pub fn set_automation_rate(&self, value: AutomationRate) {
+    pub fn set_automation_rate(&mut self, value: AutomationRate) {
```

Comment:
I don't think marking these methods as `&mut self` will work in practise, because they are return as immutable refs from the AudioNodes:
`pub fn gain(&self) -> &AudioParam`

This could of course be changed to `pub fn gain(&mut self) -> &mut AudioParam` but that is a rather big change where we should assess pros and cons.